source/includes/aws/_buckets.md

```diff
@@ -72,6 +73,42 @@ curl -X POST \
 | `taskId` <br/>*string*     | The task id related to the bucket creation. |
 | `taskStatus` <br/>*string* | The status of the operation.                  |
 
+
+<!-------------------- Update bucket -------------------->
+
+#### Update Bucket Permissions
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/rest/services/aws/test-env/buckets?operation=edit_bucket_privileges"
```

Comment:
is the bucket name different through out all regions?

Reply:
we should pass the bucket id?

Reply:
57384c4